cmd/brew-test-bot.rb

```diff
@@ -1247,12 +1300,16 @@ def test_ci_upload(tap)
       puts "brew bottle --merge --write $JSON_FILES"
     end
 
-    # These variables are for Jenkins, Circle CI, and Travis CI respectively.
+    # These variables are for Jenkins and Circle CI, and Travis CI respectively.
     upstream_number = ENV["UPSTREAM_BUILD_NUMBER"] || ENV["CIRCLE_BUILD_NUM"] || ENV["TRAVIS_BUILD_NUMBER"]
+	# If the GITHUB_TOKEN variable is set, use token+https instead of ssh for pushing.
+	# Note that we do not use 'tap.user' but instead HOMEBREW_GIT_NAME. This is because
+	# we may push to a different place than the 'tap.user' repo. For example,
+	# for Homebrew-core, tap.user=Homebrew but HOMEBREW_GIT_NAME=BrewTestBot.
     remote = if github_token
-      "https://#{github_token}@github.com/#{tap.user}/homebrew-#{tap.repo}.git"
+      "https://#{github_token}@github.com/#{HOMEBREW_GIT_NAME}/homebrew-#{tap.repo}.git"
     else
-      "git@github.com:#{tap.user}/homebrew-#{tap.repo}.git"
+      "git@github.com:#{HOMEBREW_GIT_NAME}/homebrew-#{tap.repo}.git"
```

Comment:
This is where I had to replace `tap.user` by `HOMEBREW_GIT_NAME ` in order to make sure the Linuxbrew/test-bot and Homebrew/test-bot work as expected.
Is it OK?